tests/solidity/init-node.sh

```diff
@@ -68,12 +68,12 @@ echo "$USER4_MNEMONIC" | evmd keys add "$USER4_KEY" --recover --keyring-backend
 # Set moniker and chain-id for Cosmos EVM (Moniker can be anything, chain-id must be an integer)
 evmd init "$MONIKER" --chain-id "$CHAINID" --home "$CHAINDIR"
 
-# Change parameter token denominations to aatom
-jq '.app_state["staking"]["params"]["bond_denom"]="aatom"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-jq '.app_state["gov"]["deposit_params"]["min_deposit"][0]["denom"]="aatom"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-jq '.app_state["gov"]["params"]["min_deposit"][0]["denom"]="aatom"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-jq '.app_state["evm"]["params"]["evm_denom"]="aatom"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-jq '.app_state["mint"]["params"]["mint_denom"]="aatom"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
+# Change parameter token denominations to utest
```

Comment:
if this is still a cointype 60 chain then the denom should be in `atest` and not `utest`

Reply:
fixed in a later pr